_docs/individual-payments.md

```diff
@@ -430,6 +430,28 @@ orange
 
 {% include language-tabbar.html prefix="xof-mobile-providers" raw=data-raw %}
 
+The valid `mobile_provider` values for Burkina Faso are:
+
+{% capture data-raw %}
+```
+orange
+mobicash
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xof-mobile-providers" raw=data-raw %}
+
+The valid `mobile_provider` values for Mali are:
+
+{% capture data-raw %}
+```
+orange
+mobicash
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xof-mobile-providers" raw=data-raw %}
+
```

Comment:
Here we should then have the list of possible transfer reasons:
```suggestion
{% include corridors/transfer_reasons.md %}
```
(This `include` clause points to the [full list](https://github.com/transferzero/api-documentation/blob/6f4fda6cd1f1ac9f381dc044ab8f8cbe86f68c76/_includes/corridors/transfer_reasons.md))